.github/workflows/actions.yml

```
@@ -127,3 +129,71 @@ jobs:
         grunt test
       env:
         CI: true
+  publish:
+    name: Publish to PyPI registry
+    needs:
+      - build_python
+      - build_javascript
+    runs-on: ubuntu-latest
+
+    env:
+      PY_COLORS: 1
+      TOXENV: packaging
+
+    steps:
+      - name: Switch to using Python 3.6 by default
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Install tox
+        run: python -m pip install --user tox
+      - name: Check out src from Git
+        uses: actions/checkout@v2
+        with:
+          # Get shallow Git history (default) for tag creation events
+          # but have a complete clone for any other workflows.
+          # Both options fetch tags but since we're going to remove
+          # one from HEAD in non-create-tag workflows, we need full
+          # history for them.
+          fetch-depth: >-
+            ${{
+              (
+                github.event_name == 'create' &&
+                github.event.ref_type == 'tag'
+              ) &&
+              1 || 0
+            }}
+      - name: Drop Git tags from HEAD for non-tag-create events
+        if: >-
+          github.event_name != 'create' ||
+          github.event.ref_type != 'tag'
+        run: >-
+          git tag --points-at HEAD
+          |
+          xargs git tag --delete
+      - name: Build dists
+        run: python -m tox
+      # Disabled until we get pytest-dev account maintainer on test.pypi.org
+      # - name: Publish to test.pypi.org
+      #   if: >-
+      #     (
+      #       github.event_name == 'push' &&
```

How about
```suggestion
      #       false &&
      #       github.event_name == 'push' &&
```
instead of commenting out the whole thing?